src/Wrapper.tsx

```diff
@@ -361,19 +359,24 @@ export default function Wrapper({ title, page, children }: { title: any; page: s
         <Box
           component='main'
           sx={{
-            backgroundColor: (theme) =>
-              theme.palette.mode === 'light' ? theme.palette.grey[100] : theme.palette.grey[900],
+            backgroundColor: () => (theme.palette.mode === 'light' ? theme.palette.grey[100] : theme.palette.grey[900]),
```

Comment:
Function definition unnecessary, just put the returned value directly in here.

Reply:
Done